lib/policy_machine.rb

```diff
@@ -69,15 +70,16 @@ def remove_link(src_policy_element, dst_policy_element)
   end
 
   ##
-  # Add an association between a user_attribute, an operation_set and an object_attribute
+  # Add an association between a user_attribute, an set_of_operation_objects and an object_attribute
   # in this policy machine.
   #
-  def add_association(user_attribute_pe, operation_set, object_attribute_pe)
+  def add_association(user_attribute_pe, set_of_operation_objects, operation_set, object_attribute_pe)
```

Comment:
I maintain `operation_objects` > `set_of_operation_objects`. @HonoreDB what say you?